interfaces/IBF-dashboard/src/app/services/event.service.ts

```diff
@@ -358,4 +358,29 @@ export class EventService {
       : this.state.events?.filter((e: EventSummary) => e.thresholdReached)
           .length > 0;
   }
+
+  public isLastModelDateStale = (
```

Comment:
is it also helpful to indicate next to the date how often a pipeline is expected to run? it may not be obvious why a date is red for one country-hazard but the same date is not red for a different country-hazard.